documents/EvaluateLLMJudge.md

```diff
@@ -0,0 +1,47 @@
+
+Developing specific criteria to grade outputs is not an easy task and [Who Validates the Validators? Aligning LLM-Assisted Evaluation of LLM Outputs with Human Preferences](https://arxiv.org/pdf/2404.12272) offers EvalGen as a workflow to support this task.  A key finding of their work is that the evaluation is not independent from the observed outputs and that criteria drifts over time occurs as users grading the outputs develop their criteria as they assess over time. 
+Overall, the metric approach is difficult and very bespoke especially for tasks where the LLM judge is attempting to assess none prescribed outputs or when the judging has a subjective component.    For cases where the assessment is objective and can be simplified down to a binary classification task then Cohen’s Kappa is suitable.
+
+## 3.	Identify strategies to improve the judge
+[Judging LLM-as-a-Judge with MT-Bench and Chatbot Arena](https://arxiv.org/pdf/2306.05685) & [JudgeLM: Fine-tuned large language models are scalable judges](https://arxiv.org/pdf/2310.17631) identify a range of potential biases that judges can exhibit.   These include:
+
+-	Position bias (**DN: MORE**
+-	Verbosity bias (favouring longer more verbose responses)
+-	Self-Enhancement bias (prefer answers generated by themselves)
+-	Knowledge Bias (**DN: MORE**
+
+[JudgeLM: Fine-tuned large language models are scalable judges](https://arxiv.org/pdf/2310.17631) goes on to recommend mitigations for each of these biases:
+
+-	Swap Augmentation used to mitigate position bias
+-	Reference Support to mitigate Knowledge Bias
+-	Reference Drop to mitigate format bias
+
+These three mitigations are shown to enhance the judge of interest.
+
+## Additional Considerations
+[When combinations of humans and AI are useful: A systematic review and meta-analysis](https://www.nature.com/articles/s41562-024-02024-1) finds an interesting conclusion from a systematic review of 106 studies for performance of evaluations by humans-alone vs AI alone vs human-AI combinations.   It summarised that Human-ai performed worse than the best of human alone or ai alone, but when humans alone outperformed AI alone, they found performance gains in the combination but when AI outperformed humans alone, they found losses.   The also found a trend of negative performance for human-ai synergy for classification tasks whilst a positive uplift in performance for creative tasks.
```

Comment:
I found this a little confusing:
 but when humans alone outperformed AI alone, they found performance gains in the combination but when AI outperformed humans alone, they found losses.